indico/modules/events/registration/controllers/management/fields.py

```diff
@@ -225,6 +225,7 @@ def _process(self):
         if not enabled and self.field.condition_for:
             raise NoReportError.wrap_exc(BadRequest(_('Fields used as conditional cannot be disabled')))
         self.field.is_enabled = enabled
+        signals.event.registration_form_field_changed.send(self.field)
```

Comment:
Not sure if this signal is correct here, enabling/disabling a field feels somewhat different than changing its config. I'd rather call a separate `registration_form_field_toggled`signal instead.